sympy/printing/pycode.py

```diff
@@ -185,6 +185,13 @@ def _expand_reduce_binary_op(self, op, args):
                 self._expand_reduce_binary_op(args[Nhalf:]),
             )
 
+    def _get_letter_generator_for_einsum(self):
+        for i in range(97, 123):
+            yield chr(i)
+        for i in range(65, 91):
+            yield chr(i)
```

Comment:
```suggestion
        yield from string.ascii_lowercase + string.ascii_uppercase
```

(and `import string`) above

Reply:
Ah, now I see this code just moved, so fine to leave as is.